uui/components/inputs/Checkbox.module.scss

```diff
@@ -35,10 +35,13 @@
         background-color: var(--uui-checkbox-bg);
         border: 1px solid var(--uui-checkbox-border);
         border-radius: var(--uui-checkbox-border-radius);
+
+        &:focus-within {
+            @include focus-visible-effect();
```

Comment:
I guess we don't need outline focus for checkbox, because it's can have focus via border coloring, like in TextInput.

Reply:
As we discussed with the team Checkbox, RadioInput, Switch should have outline focus.